examples/py_proto_library/example.com/proto/BUILD.bazel

```diff
@@ -0,0 +1,14 @@
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("@rules_python//python:proto.bzl", "py_proto_library")
+
+py_proto_library(
+    name = "pricetag_proto_py_pb2",
+    visibility = ["//visibility:public"],
+    deps = [":pricetag_proto"],
+)
+
+proto_library(
+    name = "pricetag_proto",
+    srcs = ["pricetag.proto"],
+    strip_import_prefix = "/example.com",
```

Comment:
nit: Could you add a comment pointing the users to the docs for `strip_import_prefix` please? E.g. I personally am not sure why we need `/example.com` instead of `example.com`.